Wireframe/index.html

```diff
@@ -1,13 +1,81 @@
+                <h2>What is Git?</h2>
+                <p>Git is a distributed version control system. It empowers developers to track changes to their code, create branches for different features or bug fixes, and seamlessly merge changes. Let's explore its essential concepts and how it fits into your workflow.</p>
+            </article>
+
+            <!-- Article 2: Why developers need Git -->
+            <article id="question2" class="second">
+                <h2>Why Git?</h2>
+                <h3>Why do developers need Git?</h3>
+                <p>Version control, a fundamental practice in software development, offers numerous advantages that streamline the development process and elevate the quality of your projects. Git and GitHub are at the forefront of these advantages.</p>
+                <div class="container">
+                    <ul class="content">
+                        <li>History Tracking</li>
+                        <li>Team Collaboration</li>
+                        <li>Conflict Resolution</li>
+                        <li>Code Quality</li>
+                        <li>Backup and Disaster Recovery</li>
+                        <li>Experimentation</li>
+                        <li>Accountability</li>
+                        <li>Efficient Collaboration</li>
+                        <li>Stable Releases</li>
+                        <li>Custom Workflows</li>
+                        <li>Remote Access</li>
+                        <li>Continuous Integration</li>
+                    </ul>
+                </div>
+            </article>
+
+            <!-- Article 3: What is a branch in Git -->
+            <article id="question3" class="last">
+                <h2>Git Branch</h2>
+                <h3>What is a branch in Git?</h3>
+                <p>In Git, a branch is similar to a separate line of development. It allows you to work on new features, bug fixes, or experiments without affecting the main codebase. Branching creates a new branch based on your current branch (usually the main branch). You can think of it as creating a parallel universe for your work.</p>
+                <p>Branching is especially useful for collaborative projects. Each developer can work on their branch, making changes independently.</p>
+            </article>
+        </div>
+
+        <!-- Summary section -->
+        <section>
+            <h2>Summary</h2>
+            <ul>
+                <li>Git is a version control system that tracks changes in your project over time.</li>
+                <li>Understanding Git terminology and the basic workflow is essential.</li>
+                <li>Common Git commands enable you to create, commit, and share code.</li>
+                <li>Branches help you work on different features simultaneously.</li>
+                <li>Git preserves a comprehensive version history for your project.</li>
+            </ul>
+        </section>
+    </main>
+    <script src="script.js"></script>
```

Comment:
What is this line doing? This task should be HTML/CSS only :)

Reply:
This line is unnecessary. And is now deleted as no JS used on this file. Only HTML & CSS advised to use on this file. Thank you for your review.